src/internal/base-component/index.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export { useComponentMetadata, COMPONENT_METADATA_KEY } from './component-metadata';
+export { useComponentMetrics } from './component-metrics';
+export { initAwsUiVersions } from './init-awsui-versions';
+export { formatMajorVersionForMetricDetail, formatVersionForMetricName } from './metrics/formatters';
+export { Metrics, MetricsTestHelper } from './metrics/metrics';
```

Comment:
`MetricsTestHelper` can stay unexported, because it is only used in unit tests

Reply:
What is the alternative then?

Reply:
```suggestion
export { Metrics } from './metrics/metrics';
```

Reply:
I mean, the `MetricsTestHelper` is needed for that test in components still, what would be the replacement?

Reply:
Found one test using it: https://github.com/cloudscape-design/components/blob/main/src/__tests__/use-telemetry-usage.test.tsx

The stateless part should move into this package. The components test should only test that this specific component calls metrics